src/main.rs

```diff
@@ -3,24 +3,24 @@ use reqwest::Client;
 use serde_json::json;
 use std::env;
 
-fn main() ->  Result<(), Err()> {
-    struct User {
-        name: String,
-        email: String
-    }
+struct User {
+    name: String,
+    email: String,
+}
```

Comment:
just a preference, can be kept inside the main function

Reply:
Sweet! 👍🏽